pkgs/applications/audio/soundconverter/default.nix

```diff
@@ -67,6 +67,11 @@ python3Packages.buildPythonApplication rec {
     runHook postCheck
   '';
 
+  postInstall = ''
+    wrapProgram $out/bin/soundconverter \
+      --prefix PYTHONPATH : "$PYTHONPATH:$out/lib:"
+  '';
```

Comment:
This looks fishy. Are executing wrapPython correctly?

Reply:
You're right, it is fishy, as I'm not 100% I did it right. But if I remove it this happens:
```
~/work/nixpkgs soundconverter-4.0.4*
 > $(nix-build -A soundconverter .)/bin/soundconverter

** (gst-plugin-scanner:220943): CRITICAL **: 10:35:01.553: pygobject initialization failed

** (gst-plugin-scanner:220943): CRITICAL **: 10:35:01.568: pygobject initialization failed
Traceback (most recent call last):
  File "/nix/store/z4ddvskny2v2pq09i5v0gjlyga5qvk98-soundconverter-4.0.4/bin/..soundconverter-wrapped-wrapped", line 94, in <module>
    from soundconverter.interface.preferences import rates
ImportError: cannot import name 'rates' from 'soundconverter.interface.preferences' (/etc/profiles/per-user/jakubgs/lib/python3.10/site-packages/soundconverter/interface/preferences.py)
```
It tries to import the library from `4.0.3` installed in my profile, which is wrong.
Suggestions as to how to fix it in a kosher way?

Reply:
Is bin/soundconverter only the gapps wrapper? They should be combined into one with wrapperArgs

Reply:
I'm not sure what you mean. Sorry, I don't know what `gapps wrapper` is.

Reply:
Is there anything else I can do to improve this?